packages/css-data/src/property-parsers/box-shadow.test.ts

```diff
@@ -0,0 +1,73 @@
+import { describe, expect, test } from "@jest/globals";
+
+import { parseBoxShadow } from "./box-shadow";
+
+describe("parseBoxShadow", () => {
```

Comment:
describe here not necessary

Reply:
we have always been wrapping a tests file into describe, so far, not sure what's the difference in logging output